### PR TITLE
Rename Mixed Reality Toolkit menus to Mixed Reality

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change renames the "Mixed Reality Toolkit" menus with "Mixed Reality" to unify with other packages (ex: OpenXR extensions)

Top level:
image

Assets menu:
image

### CODE REVIEW

File: Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Microsoft.MixedReality.Toolkit.Boundary
     /// <summary>
     /// Configuration profile settings for setting up boundary visualizations.
     /// </summary>
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Mixed Reality Boundary Visualization Profile", fileName = "MixedRealityBoundaryVisualizationProfile", order = (int)CreateProfileMenuItemIndices.BoundaryVisualization)]
+    [CreateAssetMenu(menuName = "Mixed Reality/Profiles/Mixed Reality Boundary Visualization Profile", fileName = "MixedRealityBoundaryVisualizationProfile", order = (int)CreateProfileMenuItemIndices.BoundaryVisualization)]
```
Comment (keveleigh) [inline Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs:13]: We should still group these under a "Toolkit" heading, so we don't clutter the top level. Either `Mixed Reality/Toolkit` or `Mixed Reality/Mixed Reality Toolkit` or maybe `Mixed Reality/MRTK`, but it should definitely be grouped imo

**Review by keveleigh — CHANGES_REQUESTED** [review]

File: Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs:13]: Sounds reasonable. Please note that making the suggested change will deviate from the top level menu (unless we add a Toolkit or MRTK mid level menu there as well)

File: Assets/MRTK/Core/Definitions/CameraSystem/MixedRealityCameraProfile.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+// Licensed under the MIT License.``
```
Comment (keveleigh) [inline Assets/MRTK/Core/Definitions/CameraSystem/MixedRealityCameraProfile.cs:2]:
```suggestion
// Licensed under the MIT License.
```

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs [hunk 3]
```diff
@@ -48,7 +48,7 @@ public override void Initialize()
         {
             base.Initialize();
             UnityEngine.Debug.Log(@"Detected a potential deployment issue for the Oculus Quest. In order to use hand tracking with the Oculus Quest, download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
-The tool can be found under <i>Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules</i>");
+The tool can be found under <i>Mixed Reality > Utilities > Oculus > Integrate Oculus Integration Unity Modules</i>");
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:51]:
```suggestion
The tool can be found under <i>Mixed Reality > Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules</i>");
```

Comment (davidkline-ms) [reply Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs]: thanks for the catch!

**Review by keveleigh — APPROVED** [review]
Are you planning on updating the docs now too? Or should we file a follow-up task?

**Comment by davidkline-ms** [comment]
> Are you planning on updating the docs now too? Or should we file a follow-up task?

Since docs are in a different repo, i was planning on doing it separately (in the next day or so).

**Pull request merged**